lib/phoenix/live_dashboard/helpers.ex

```diff
@@ -1,22 +1,28 @@
 defmodule Phoenix.LiveDashboard.Helpers do
   @moduledoc false
 
-  alias Phoenix.LiveDashboard.PageBuilder
+  alias Phoenix.LiveDashboard.{PageBuilder, SystemInfo}
   import Phoenix.LiveView.Helpers
   @format_limit 100
 
   @doc """
   Formats any value.
   """
-  def format_value(port, live_dashboard_path) when is_port(port) do
-    live_patch(inspect(port),
-      to: live_dashboard_path.(node(port), info: PageBuilder.encode_port(port))
```

Comment:
We should also leave the previous clause (and make sure it is tested) because custom pages may handling ports and pids too.